sklearn/utils/tests/test_utils.py

```diff
@@ -464,6 +466,22 @@ def test_safe_indexing_container_axis_0_unsupported_type():
         _safe_indexing(array, indices, axis=0)
 
 
+def test_safe_indexing_pandas_no_settingwithcopy_warning():
+    # Using safe_indexing with an array-like indexer gives a copy of the
+    # DataFrame -> ensure it doesn't raise a warning if modified
+    pd = pytest.importorskip("pandas")
+    if parse_version(pd.__version__) < parse_version("0.25.0"):
+        raise SkipTest(
+            "Older pandas version will raise a different warning than "
+            "SettingWithCopyWarning"
```

Comment:
```suggestion
            "Older pandas version still raise a SettingWithCopyWarning warning"
```

(it's not a different warning, but it's still raising one (the test is testing it doesn't raise any warning))

Reply:
Right